sys/include/ubjson.h

```diff
@@ -20,12 +20,16 @@
  * @defgroup sys_ubjson  Universal Binary JSON library
  * @ingroup  sys_serialization
  * @brief    Provides a library to read and write UBJSON serialized data
+ *
+ * @deprecated  This module is deprecated and will be removed in future releases.
```

Comment:
Also it should be specified which release at the earliest (usually 2 releases in the future, since a release is imminent I would say 2020.01 though) and given that we have two (`tinycbor` and `cn-cbor`) we should mention alternatives to port to.

Reply:
Agreed on the 2020.01 thing. Regarding alternatives, {x}cbor may or may not be an alternative (it parses another format) so I don't think it is of much help to mention it.

Reply:
They are not compatible, but it may help someone looking for an alternative perhaps.

Reply:
If it will become a matter of contention I will add it. I don't want to do it because:

- It is not up to us to tell people how to develop their apps.
- I wouldn't expect users to switch serialization formats - at least, it is not something you decide in a couple of minutes reading a compiler warning.
- CBOR is already mentioned in the lost+found and in the commit message for the removal.

Reply:
That's fine with me, I won't push for this.

Reply:
I know it's not compatible. I just think its nice to offer alternatives when something is removed. You don't have to _tell_ people, but it could be suggestion ;-).

> * CBOR is already mentioned in the lost+found and in the commit message for the removal.

Ok fine.